cmd/minishift/cmd/start.go

```diff
@@ -231,6 +231,7 @@ func postClusterUp(machineName string, ip string, port int, routingSuffix string
 		atexit.Exit(1)
 	}
 
+	configurePeristentVolumes(sshCommander)
```

Comment:
I think it is time to raise an issue with cluster up to create only 10 directories and provide a flag to allow users to create more if user want. @hferentschik you might have raised an issue for this. Nor sure.